src/tests/ftest/control/daos_control_config.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+from apricot import TestWithServers
+from command_utils_base import CommandFailure
+
+
+class DaosControlConfigTest(TestWithServers):
```

Comment:
Why is this test called DaosControlConfigTest? You're checking if dmg storage scan works with given config file, so isn't it more like DmgConfigTest?

Reply:
That's a good point. I called it control config test to keep consistency with the dmg config name which is daos_control.yaml

Reply:
I see. I didn't know that was the name of the config file.

Reply:
Yeah, FYI... The file was renamed from `daos.yml` to `daos_control.yml` in order to better reflect its relationship to what it's configuring. The dmg tool uses the Control API, but the expectation is that the new API may be used to build other things.

Reply:
I see. Thanks.